src/Payload/Level.php

```diff
@@ -2,6 +2,17 @@
 
 class Level implements \JsonSerializable
 {
+    const IGNORE = 'ignore';
```

Comment:
`IGNORE` and `IGNORED` are synonymous and they can be used to completely silence the logger if used in `minimumLevel` config option. I think it's mostly a leftover from 0.18.2, I don't really see the point and it's also missing from README.md. I highly doubt anybody is using them so I'm happy to remove them.

`ALERT` and `EMERGENCY` are PSR-3 levels and are mapped to Rollbar levels here: https://github.com/rollbar/rollbar-php/blob/master/src/Payload/Level.php#L16-L33